test/models/claim.test.js

```diff
@@ -0,0 +1,10 @@
+describe('Test Claim model', () => {
+  test('claim model is created', async () => {
+    jest.mock('sequelize', () => {
+      const mockSequelize = require('sequelize-mock')
+      return mockSequelize
+    })
+    const claimModel = require('../../server/models/claim')
+    expect(claimModel.name).toEqual('')

```

Comment:
This doesn't seem to be testing anything useful. Models seem to be imported via `sequelize.import` in `server/models/index.js` so I think testing that a model can be created should be done via there rather than by importing the model directly. Then we'd be proving that the model definition is considered valid by Sequelize.

Reply:
The sequalize module is mocked so that it doesn't require a database connection. The mock-sequalize module doesn't seem to import the models in the same way so the code isn't included in the code coverage report. Might be something to look at later.

Reply:
I've raised a ticket to look into this properly: https://eaflood.atlassian.net/browse/FPD-374